app/src/main/kotlin/cz/covid19cz/erouska/net/FirebaseFunctionsRepository.kt

```diff
@@ -29,8 +31,8 @@ class FirebaseFunctionsRepository(
             "model" to deviceInfo.getDeviceName(),
             "locale" to LocaleUtils.getLocale()
         )
-        val ehrid = checkNotNull(callFunction("RegisterEhrid", data)["ehrid"])
-        prefsRepository.saveEhrid(ehrid)
+        val token = checkNotNull(callFunction("RegisterEhrid", data)["customToken"])
+        prefs.saveRegisterToken(token)
```

Comment:
Token není potřeba ukládat do sharedpreferences, ale prostě rovnou po registerEhrid se s ním přihlásit. Má omezenou platnost, později už to nebude fungovat. Naopak přihlášení zůstane perzistentní.